packages/rum/test/rum.spec.ts

```diff
@@ -547,23 +508,24 @@ describe('rum global context', () => {
   })
 
   it('should be removable', () => {
-    const { server, lifeCycle, rumApi } = setupBuilder.build()
+    const { server, lifeCycle, setGlobalContext } = setupBuilder.build()
     server.requests = []
 
-    rumApi.addRumGlobalContext('bar', 'foo')
+    const globalContext = { bar: 'foo' }
+    setGlobalContext(globalContext)
     lifeCycle.notify(LifeCycleEventType.ERROR_COLLECTED, FAKE_ERROR as ErrorMessage)
-    rumApi.removeRumGlobalContext('bar')
+    delete globalContext.bar
```

Comment:
what about splitting these tests in two:
- unit test of global context CRUD in rum.entry.spec
- unit test of global context attached to request / snake case in rum.spec

Reply:
I thought about it, and I think we could benefit from a small common module that would implement context handling (add/set/remove), used in logs.entry, rum.entry and logger. This way, we would only need to test this common module.

In any case, since the context is (currently) mutable, I think it makes sense to keep a test like this here, to show that the current context is deeply merged into the generated event. The test name could be more accurate though (like: `should ignore subsequent context mutation`)

WDYT?